docs/core/linux-prerequisites.md

```diff
@@ -15,38 +15,51 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 
 This article shows the dependencies needed to develop .NET Core applications on Linux. The supported Linux distributions/versions, and dependencies that follow apply to the two ways of developing .NET Core apps on Linux:
 
-* [Command line](tutorials/using-with-xplat-cli.md)
+* [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
 * [Visual Studio Code](https://code.visualstudio.com/)
 
 ## Supported Linux versions
 
-.NET Core 2.x is supported on the following Linux distributions/versions:
-
- * Red Hat Enterprise Linux 7 x64
- * CentOS 7 x64
- * Oracle Linux 7 x64
- * Fedora 25, 26 x64
- * Debian 9, 8.7+ x64 
- * Ubuntu 17.04, 16.04, 14.04  x64
- * Linux Mint 18, 17 x64
- * openSUSE 42.2+ x64
- * SUSE Enterprise Linux (SLES) 12 SP2+ x64
-
-.NET Core 1.x is supported on the following Linux distributions/versions:
-
-* Red Hat Enterprise Linux / CentOS / Oracle Linux 7 x64
-* Fedora 24 x64
-* Debian 8.2+ x64
-* Ubuntu / Linux Mint 14.04, 16.04, 16.10*, 17 x64
-* openSUSE 42.1 (1.1) x64
-> [!NOTE]
-> Ubuntu / Linux 16.10 is supported by the latest patch release of .NET Core 1.1
+# [.NET Core 2.x](#tab/netcore2x)
+
+.NET Core 2.0 treats Linux as a single operating system. There is a single Linux build (per chip architecture) for supported Linux distros.
+
+NET Core 2.x is supported on the following Linux x64 distributions/versions:
+
+ * Red Hat Enterprise Linux 7
+ * CentOS 7
+ * Oracle Linux 7
+ * Fedora 25, Fedora 26
+ * Debian 8.7 or later versions 
+ * Ubuntu 17.04, Ubuntu 16.04, Ubuntu 14.04
+ * Linux Mint 18, Linux Mint 17
+ * openSUSE 42.2 or later versions
+ * SUSE Enterprise Linux (SLES) 12 SP2 or later versions
 
 See [.NET Core 2.x Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) for the complete list of .NET Core 2.x supported operating systems, out of support OS versions, and lifecycle policy links.
 
+# [.NET Core 1.x](#tab/netcore1x)
+
+.NET Core 1.x is supported on the following Linux x64 distributions/versions:
+
+* Red Hat Enterprise Linux 7
+* CentOS 7
+* Oracle Linux 7
+* Fedora 24
+* Debian 8.2 or later versions
+* Ubuntu 14.04, Ubuntu 16.04, Ubuntu 16.10\*
+ * Ubuntu 16.10 is supported by the latest patch release of .NET Core 1.1
+* Linux Mint 17
+* openSUSE 42.1 (.NET Core 1.1)
```

Comment:
Should be "openSUSE 42.1 or later versions". We realized this week that the current language completely cut openSUSE support off when 42.1 was end-of-life'd in May.

Reply:
To clarify - .NET Core 1.1 supports openSUSE 42.1 or later versions?